lib/command/run-multiple.js

```diff
@@ -46,47 +47,59 @@ module.exports = function (suites, options) {
     suite = suite.split(':');
 
     let suiteName = suite[0];
-    let browser = suite[1];
+    let browserConfig = {browser: suite[1], windowSize: suite[2]};
     let suiteConf = configMultiple[suiteName];
 
     if (!suiteConf) {
       throw new Error(`Suite ${suiteName} was not configured in "multiple" section of config`);
     }
 
-    runSuite(suiteName, suiteConf, browser);
+    runSuite(suiteName, suiteConf, browserConfig, outputFolder, defaultWindowSize);
```

Comment:
please don't add extra params; just add output to suiteConf

```js
suiteConf.output = outputFolder
```

and keep old signature